backend/app/api/v1/users.py

```
@@ -0,0 +1,139 @@
+
+@router.get("/me", response_model=UserProfileResponse)
+async def get_current_user_profile(user_id: UUID = Depends(get_current_user)):
+    """Get the current user's profile information."""
+    try:
+        user = await get_user_by_id(str(user_id))
+        
+        if not user:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="User profile not found"
+            )
```

Ruff (0.15.4)

[warning] Abstract `raise` to an inner function

(TRY301)